mkdocs.yml

```
@@ -50,6 +50,7 @@ nav:
       - Container Environment Variables: docs/developing/environment-variables.en.md
       - Custom Environment Resources: docs/developing/custom-environment-resources.en.md
       - Domain: docs/developing/domain.en.md
+      - Internal ALBs: docs/developing/internal-albs.en.md
```

nit: Folks might not know what ALB stands for
```suggestion
      - Internal Load Balancers: docs/developing/internal-albs.en.md
```